requirements.txt

```diff
@@ -0,0 +1,20 @@
+attrs>=21.4.0
```

Comment:
Would you recommend we use `==` or `>=` in our requirements? Or just not bother for now?

Reply:
I prefer using `>=`, I remember we had this conversation before at Qxf2, we felt with the `requirements` file, we are assuring users that our code works on a module from a version to any latest.